config/additional-supervisord.conf

```diff
@@ -1,5 +1,7 @@
 [program:freshclam]
 command = freshclam -d
+# scrub potentially sensitive environment variables we don't need
+environment = AWS_ACCESS_KEY_ID="",AWS_SECRET_ACCESS_KEY="",DM_ANTIVIRUS_API_AUTH_TOKENS="",DM_ANTIVIRUS_API_CALLBACK_AUTH_TOKENS="",DM_NOTIFY_API_KEY="",DM_DEVELOPER_VIRUS_ALERT_EMAIL=""
```

Comment:
It's annoying that we have to list every variable that we don't want, rather than the ones we do want 🤔

Reply:
It _is_ but on the other hand if we had it the other way round we'd probably find ourselves having to specify all kinds of boring env vars we don't usually have to think about.

Reply:
Not that `supervisord` supports the other way around, but I'm sure patches would be welcomed...